cmd/brew-check-for-deleted-upstream-core-formulae.rb

```diff
@@ -7,36 +7,41 @@
 module Homebrew
   module_function
 
-  def homebrew_repo_dir
-    @homebrew_repo_dir ||= ARGV.value("homebrew-repo-dir") || ENV["HOMEBREW_REPO_DIR"]
-  end
-
-  def linuxbrew_repo_dir
-    @linuxbrew_repo_dir ||= ARGV.value("linuxbrew-repo-dir") || "/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core"
+  def git?
+    homebrew_repo_dir == linuxbrew_repo_dir
   end
 
   def linux_only?(formula)
-    File.open("#{linuxbrew_repo_dir}/Formula/#{formula}").grep("# tag \"linux").any?
+    File.read("#{linuxbrew_repo_dir}/Formula/#{formula}").match("# tag \"linux\"")
   end
 
-  def homebrew_core_formulae
-    @homebrew_core_formulae ||= Dir.entries("#{homebrew_repo_dir}/Formula").reject! { |f| File.directory?(f) }.sort
+  def homebrew_repo_dir
+    @homebrew_repo_dir ||= ARGV.value("homebrew-repo-dir") || ENV["HOMEBREW_REPO_DIR"] || CoreTap.instance.path
```

Comment:
```suggestion
    @homebrew_repo_dir ||= ARGV.value("homebrew-repo-dir") || CoreTap.instance.path
```

Not sure we need the envvar now - I think that's just *too* many different ways? Having thought about it some more, ,the git solution is better, and if people (erm, me) want other ways then the args are easy enough.

Reply:
Done.